### PR TITLE
Move atom library bootstrap in standard

### LINKED ISSUE #18841: Snippets atom rendering is too slow
When we [implemented rendering of snippets atom](https://github.com/guardian/frontend/pull/18222) with the [atom renderer library](https://github.com/guardian/atom-renderer) we discussed how `CSS` integration could be done:

- Include the css of the atom-type server side in a similar way that we do for [interactives atom](https://github.com/guardian/frontend/blob/e394924306592c03959678b3f03f38fdf1c03c95/common/app/views/fragments/atoms/interactive.scala.html#L39-L41) 

- Load the `CSS` on demand client side 

The solution we decided at that time, was to load the CSS client-side on demand using `webpack`. 

The problem with the current solution is that CSS loading happens at the end (**even after ads have rendered**!) and the problem has been noticed and reported by both our users and editors who are unhappy with the current outcome.

Snippets atom content is fully part of the content, so its styling need to be applied at the same time than the style of the content.

Therefore I think we need to revisit the chosen solution.  

When a piece of content is asked to be rendered, we know server side which atoms are part of the response (available in the Content API resource) so we could easily compute the `CSS` of atom types to include and include them either inlined or linked in the server response.  

## Steps to Reproduce

Open a page containing a snippet atom, [for instance](https://www.theguardian.com/us-news/2018/jan/11/trump-pans-immigration-proposal-as-bringing-people-from-shithole-countries)   

Look at the atom style, wait and notice how long it take to be rendered correctly ⌚ ⏳

## Actual Results 

screen shot 2018-01-12 at 15 50 40

## Expected Results 

screen shot 2018-01-12 at 16 10 00

@sndrs @SiAdcock @nicl @NataliaLKB @regiskuckaertz

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Fixes https://github.com/guardian/frontend/issues/18841

Will hopefully get us way closer to an acceptable UX.

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
I'm relying on `fastdom-promise` but am happy to demote to just `fastdom` and create a `Promise` myself -- I _think_ `Promise` is polyfilled before reaching that point, isn't it?

**Comment by mchv** [comment]
👍 from me but will be good to have @GHaberis or @sndrs review and think of any potential issue (as `Promise` not being polyfiled at that time)

**Comment by sndrs** [comment]
@regiskuckaertz yeah should be handled by polyfill.io

**Review by sndrs — APPROVED** [review]
lgtm 🤞

File: static/src/javascripts/projects/atoms/services.js [hunk 1]
```diff
@@ -15,11 +15,20 @@ import { viewport } from './services/viewport';
 //    ...
 // }
 
+type FastdomAction = Function => void;
+
+const promisify = (f: FastdomAction) => (g: Function): Promise<any> =>
```
Comment (sndrs) [inline static/src/javascripts/projects/atoms/services.js]: can we use fuller variable names? there's no overhead in being 100% explicit here

Comment (regiskuckaertz) [reply static/src/javascripts/projects/atoms/services.js]: yeah of course

Comment (sndrs) [reply static/src/javascripts/projects/atoms/services.js]: ta

**Pull request merged**